components/Button.vue

```diff
@@ -46,19 +46,27 @@ export default {
   font-size: 1rem;
   font-weight: bold;
   border: none;
-  margin-right: 16px;
+  margin-right: 10px;
+  margin-bottom: 5px;
   background-color: transparent;
-  border-radius: 25px;
-  padding: 12px 16px;
+  border-radius: 55px;
+  padding: 16px 38px 16px 22px;
   outline: 0;
   text-decoration: none;
   cursor: pointer;
   white-space: nowrap;
   width: fit-content;
   height: fit-content;
   box-sizing: border-box;
+  min-width: 180px;
+  position: relative;
+  text-align: left;
+  letter-spacing: -0.02em;
 
   .arrow {
+    position: absolute;
```

Comment:
The arrow should be also centered 
<img width="216" alt="Captura de pantalla 2021-04-14 a las 16 14 58" src="https://user-images.githubusercontent.com/16032491/114725333-b4b08d00-9d3c-11eb-82f8-bb749e7c83f8.png">

Reply:
It looks centered in my system 😶
![image](https://user-images.githubusercontent.com/1316439/114726519-c2b2dd80-9d3d-11eb-9406-8386b77b5be9.png)

Reply:
This is the issue preventing the buttons from looking perfect in every browser:
https://bugs.chromium.org/p/chromium/issues/detail?id=1023654